.github/workflows/pypipublish_windows.yml

```diff
@@ -0,0 +1,56 @@
+name: Windows Package github deploy
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-windows-wheel:
+
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8] # 3.5 Currently 3.5 does not run through the tests.yml CI. Leaving it out until testng is implemented.
```

Comment:
Do you mean that it doesn't support 3.5 on Windows or OSX? AFAIK it was running the tests on Linux.

Reply:
3.5 needs to be added here. It's only the black test regardless.

Reply:
this needs to be done